components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -322,15 +317,10 @@ func main() {
 	router.Handle("/info/runtimes", runtimesInfoHandler)
 
 	// create OSB API endpoints
-	basicAuth := &broker.Credentials{
-		Username: cfg.Auth.Username,
-		Password: cfg.Auth.Password,
-	}
 	router.Use(middleware.AddRegionToContext(cfg.DefaultRequestRegion))
 	for prefix, creds := range map[string]*broker.Credentials{
-		"/":                basicAuth, // legacy basic auth
-		"/oauth/":          nil,       // oauth2 handled by Ory
-		"/oauth/{region}/": nil,       // oauth2 handled by Ory with region
+		"/oauth/":          nil, // oauth2 handled by Ory
+		"/oauth/{region}/": nil, // oauth2 handled by Ory with region
 	} {
 		route := router.PathPrefix(prefix).Subrouter()
 		broker.AttachRoutes(route, kymaEnvBroker, logger, creds)
```

Comment:
this can be now simplified. Please remove the `creds` param and basic auth middleware from `broker.AttachRoutes`.

Please also remove the `broker.Credentials` struct. 


final statement:

```go
	for _, prefix := range []string{
		"/oauth/",          // oauth2 handled by Ory
		"/oauth/{region}/", // oauth2 handled by Ory with region
	} {
		route := router.PathPrefix(prefix).Subrouter()
		broker.AttachRoutes(route, kymaEnvBroker, logger)
	}
```